gtsfm/bundle/bundle_adjustment.py

```diff
@@ -54,8 +54,6 @@
 
 # noise model params
 CAM_POSE3_PRIOR_NOISE_SIGMA = 0.1
```

Comment:
could add a parameter for this as well?

Reply:
actually, on second thought, not really required, we dont use pose priors for any current datasets. 
measurement_noise_sigma might be more important.

Reply:
I have added both.